crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -90,6 +136,20 @@ pub enum EnvMode {
     Strict,
 }
 
+impl fmt::Display for EnvMode {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "{}",
+            match self {
+                EnvMode::Infer => "infer",
+                EnvMode::Loose => "loose",
+                EnvMode::Strict => "strict",
+            }
+        )
+    }
```

Comment:
We can avoid template string literal if we invoke [`write_str`](https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.write_str) directly.
```suggestion
        f.write_str(
            match self {
                EnvMode::Infer => "infer",
                EnvMode::Loose => "loose",
                EnvMode::Strict => "strict",
            }
        )
    }
```